apps/nextjs-website/src/app/profile/personal-data/page.tsx

```diff
@@ -0,0 +1,73 @@
+'use client';
+
+import { Stack, Typography } from '@mui/material';
+import { useTranslations } from 'next-intl';
+import { useUser } from '@/helpers/user.helper';
+import { InfoCardItemProps } from '@/components/atoms/InfoCardItem/InfoCardItem';
+import { InfoCard } from '@/components/molecules/InfoCard/InfoCard';
+import PageNotFound from '@/app/not-found';
+
+const TermsOfService = () => {
+  const t = useTranslations('profile');
+  const { user, loading } = useUser();
+
+  const dataSectionItems: InfoCardItemProps[] = [
+    {
+      title: t('personalData.fields.name'),
+      value: user?.attributes.given_name,
+    },
+    {
+      title: t('personalData.fields.surname'),
+      value: user?.attributes.family_name,
+    },
+    {
+      title: t('personalData.fields.role'),
+      value: user?.attributes['custom:job_role'],
+    },
+    {
+      title: t('personalData.fields.company'),
+      // TODO: add company when available
+    },
+    {
+      title: t('personalData.fields.sector'),
+      value: user?.attributes['custom:company_type'],
+    },
+    {
+      title: t('personalData.fields.products'),
+      // TODO: add products when available
+    },
+  ];
+
+  const accountSectionItems: InfoCardItemProps[] = [
+    {
+      title: t('personalData.fields.email'),
+      value: user?.attributes.email,
+    },
+    {
+      title: t('personalData.fields.password'),
+      value: '••••••••••••',
+    },
+  ];
+
+  // TODO: add dedicated loading and unauthorized pages
+  if (!loading && !user) {
+    return <PageNotFound />;
+  }
+
+  return (
+    <Stack gap={5} sx={{ padding: 5, width: '100%' }}>
+      <Typography variant='h3'>{t('personalData.title')}</Typography>
+      <InfoCard
+        cardTitle={t('personalData.dataSection')}
+        items={dataSectionItems}
+      />
+
+      <InfoCard
+        cardTitle={t('personalData.accountSection')}
+        items={accountSectionItems}
+      />
+    </Stack>
```

Comment:
You should add the "delete user" section (see this [link](https://docs.aws.amazon.com/cognito-user-identity-pools/latest/APIReference/API_DeleteUser.html) for the doc)

Reply:
The deleted section is realtive to a different [task](https://pagopa.atlassian.net/jira/software/projects/DEV/boards/280?selectedIssue=DEV-1020) and will be added in a separte PR.